src/main/java/com/amazon/janusgraph/diskstorage/dynamodb/iterator/MultiRowParallelScanInterpreter.java

```diff
@@ -68,7 +68,7 @@ public MultiRowParallelScanInterpreter(DynamoDBStore store, SliceQuery sliceQuer
      * Because we are scanning in segments across the entire hash key space, it is possible for the same hash key to appear in two different segments.
      * We are also running all of the scan segments in parallel, so we have no control over which segment returns first.
      *
-     * In the example, if segment 2 was the first segment to post a result, we would store hk:B as a "boundary" key. That way when
```

Comment:
Not sure why this says there is a change; it looks the same as [the original](https://github.com/awslabs/dynamodb-janusgraph-storage-backend/blob/master/src/main/java/com/amazon/janusgraph/diskstorage/dynamodb/iterator/MultiRowParallelScanInterpreter.java#L71).